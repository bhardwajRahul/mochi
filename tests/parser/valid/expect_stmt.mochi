let x = 2
expect x > 1
