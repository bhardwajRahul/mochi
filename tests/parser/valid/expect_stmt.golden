(program
  (let x (int 2))
  (expect
    (binary > (selector x) (int 1))
  )
)
